draft-ietf-scitt-architecture.md

```diff
@@ -175,6 +177,158 @@ SCITT achieves this by having producers publish information in a Transparency Se
+In consequence, instead of trying to understand and describe the detailed syntax and semantics of every type of statement about digital products, the SCITT architecture focuses on ensuring statement authenticity, visibility/transparency, and intends to provide scalable accessibility.
+The following use cases illustrate the scope of SCITT and elaborate on the generic problem statement above.
+
+## Eclectic SSC Use Cases
+
+The three following use cases are a specialization derived from the generic problem statement above.
+These three use case and more were incubated and selected from this Internet-Draft: {{-all-scitt-use-cases}}.
+
+### Security Analysis of a Software Product
+
+A released software product is often accompanied by a set of complementary statements about it's security compliance.
+This gives enough confidence to both producers and consumers that the released software has a good security standard and is suitable to use.
+
+Subsequently, multiple security researchers often run sophisticated security analysis tools on the same product.
+The intention is to identify any security weaknesses or vulnerabilities in the package.
+
+Initially, a particular analysis can identify a simple weakness in a software component.
+Over a period of time, a statement from a third-party illustrates that the weakness is exposed in a way that represents an exploitable vulnerability.
+The producer of the software product provides a statement that confirms the linking of software component vulnerability with the software product and also issues an advisory statement on how to mitigate the vulnerability.
+At first, the producer provides an updated software product that still uses the vulnerable software component but shields the issue in a fashion that inhibits exploitation.
+Later, a second update of the software product includes a security patch to the affected software component from the software producer.
+Finally, a third update includes a new release (updated version) of the formerly insecure software component.
+For this release, both the software product and the affected software component are deemed secure by the producer and consumers.
+
+A consumer of a released software wants to:
+
+* know where to get these security statements from producers and third-parties related to the software product in a timely and unambiguous fashion
+* attribute them to an authoritative issuer
+* associate the statements in a meaningful manner via a set of well-known semantic relationships
+* consistently, efficiently, and homogeneously check their authenticity
+
+Without SCITT there would be no standardized way to:
+
+* know the various sources of statements
+* express the provenance and historicity of statements
+* relate and link various heterogeneous statements in a simple fashion
+* check that the statement comes from a source with authority to issue that statement
```

Comment:
* confirm that sources provide a complete history of statements related to a given component